### DISCUSSION TITLE
Question about capturing iframe events for play button clicks

### DISCUSSION THREAD

**Discussion by ingcol** [datarhei/restreamer#782] [General]
Hello,

I hope you are well. I am currently using the service with you, in which I configured a subdomain, and we use this embedded in an iframe of a vue application. My goal is to track when users click the play button in the integrated service and send a POST request to my server. However, due to browser security restrictions, I cannot directly access the events within the iframe.

Could you please let us know if there is a recommended approach or API method within Restreamer that allows capturing events such as play button clicks within the iframe? Alternatively, is there a way to communicate between the iframe and the main Vue.js application to achieve this functionality?

Any guidance or suggestions will be greatly appreciated. Thanks for your support!

**Comment by ioppermann** [comment]
Instead of iframe-ing the Restreamer player, you could integrate a player natively (e.g. https://videojs.com/guides/vue/) into your app and provide it with the HLS URL from Restreamer. With this approach you will not have to deal with iframes anymore and it will be likely possible to capture the events.

**Reply by ingcol** [reply]
Thanks for answering. But in my case the configuration is on a subdomain and the url obtained is of the type https://sudomain/800a956d-5ada-4bf1-ac15-0d11e689179c.html, so it cannot be reproduced in a video tag, therefore it is used in the iframe
